<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>students</class>
 <widget class="QMainWindow" name="students">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>528</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>737</width>
    <height>528</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>553</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Student</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ssl.qrc">
    <normaloff>:/s.png</normaloff>:/s.png</iconset>
  </property>
  <widget class="QWidget" name="s_centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="s_tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="s_classes">
       <attribute name="title">
        <string>Classes</string>
       </attribute>
       <widget class="QListWidget" name="classes_listWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>411</width>
          <height>361</height>
         </rect>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Classes</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="s_grades">
       <attribute name="title">
        <string>Grades</string>
       </attribute>
       <widget class="QComboBox" name="gradesSubject_comboBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>241</width>
          <height>24</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Subject</string>
        </property>
       </widget>
       <widget class="QComboBox" name="testOrAss_comboBox_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>80</y>
          <width>72</width>
          <height>24</height>
         </rect>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Choose</string>
        </property>
        <item>
         <property name="text">
          <string>Test</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Assignment</string>
         </property>
        </item>
       </widget>
       <widget class="QTableView" name="grades_tableView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>401</width>
          <height>271</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
       <widget class="QPushButton" name="gradesGet_pushButton">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>80</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Get</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="s_assignments">
       <attribute name="title">
        <string>Assignments</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>711</width>
          <height>531</height>
         </rect>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="s_assDownload">
         <attribute name="title">
          <string>Download</string>
         </attribute>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>661</width>
            <height>481</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>659</width>
             <height>479</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="s_assSubmit">
         <attribute name="title">
          <string>Submit</string>
         </attribute>
         <widget class="QComboBox" name="submit_comboBox_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>371</width>
            <height>24</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>Subject - Assignment Title</string>
          </property>
         </widget>
         <widget class="QToolButton" name="assignmentfile_toolButton">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>200</y>
            <width>23</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="assignmentPath_lineEdit_10">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>200</y>
            <width>231</width>
            <height>24</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>Upload File...(PDF,DOCS)</string>
          </property>
         </widget>
         <widget class="QPushButton" name="assignmentSubmit_pushButton_3">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>320</y>
            <width>80</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="s_profile">
       <attribute name="title">
        <string>My Profile</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>91</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Age:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="profileAge_lineEdit_6">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>70</y>
          <width>361</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="profileName_lineEdit_8">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>150</y>
          <width>201</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Student Id:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="profileID_lineEdit_7">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="profileImage_label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid black;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>img</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="commandLinkButton">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>430</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Log Out</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../Downloads/logout.png</normaloff>../../../../Downloads/logout.png</iconset>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="s_messages">
       <attribute name="title">
        <string>Messages</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>New Message</string>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>661</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>611</width>
          <height>471</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>609</width>
           <height>469</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ssl.qrc"/>
 </resources>
 <connections/>
</ui>
